### ISSUE TITLE
the example of science/cover get Error: wrong path

### ISSUE THREAD

**Issue by Lyoko-Jeremie** [DIYgod/RSSHub#4173]
### 路由地址

https://rsshub.app/science/cover

### 预期是什么？

this link come from the document:
https://docs.rsshub.app/journal.html#feng-mian-gu-shi-2

### 实际发生了什么？

```
Looks like something went wrong

Error: wrong path
    at module.exports (/data/wwwroot/rsshub.app/lib/middleware/parameter.js:14:15)
    at runMicrotasks ()
    at processTicksAndRejections (internal/process/task_queues.js:94:5)
    at async module.exports (/data/wwwroot/rsshub.app/lib/middleware/template.js:15:5)
    at async /data/wwwroot/rsshub.app/lib/middleware/api-response-handler.js:143:9
    at async module.exports (/data/wwwroot/rsshub.app/lib/middleware/api-template.js:2:5)
    at async module.exports (/data/wwwroot/rsshub.app/lib/middleware/utf8.js:4:5)
    at async module.exports (/data/wwwroot/rsshub.app/lib/middleware/debug.js:14:5)
    at async module.exports (/data/wwwroot/rsshub.app/lib/middleware/access-control.js:29:13)
    at async module.exports (/data/wwwroot/rsshub.app/lib/middleware/header.js:20:5)
    at async module.exports (/data/wwwroot/rsshub.app/lib/middleware/onerror.js:20:9)
```

### 额外信息（日志、报错等）

maybe the documents wrong or outdate ?

**Comment by hoilc** [comment]
@yech1990 期刊文档有个typo，应为`/sciencemag/cover`

**Comment by yech1990** [comment]
fixed in https://github.com/DIYgod/RSSHub/pull/4159

sorry about that..

**Comment by hoilc** [comment]
Closed by #4159
